### PR TITLE
Octave kernel

### PR DESCRIPTION
**Pull request by diocas**
Backport from master

### CODE REVIEW

File: systemuser.sh [hunk 1]
```diff
@@ -48,6 +48,7 @@ JPY_CONFIG=$JUPYTER_CONFIG_DIR/jupyter_notebook_config.py
 echo "c.FileCheckpoints.checkpoint_dir = '$SCRATCH_HOME/.ipynb_checkpoints'"         >> $JPY_CONFIG
 echo "c.NotebookNotary.db_file = '$JUPYTER_LOCAL_PATH/nbsignatures.db'"     >> $JPY_CONFIG
 echo "c.NotebookNotary.secret_file = '$JUPYTER_LOCAL_PATH/notebook_secret'" >> $JPY_CONFIG
+echo "c.NotebookApp.iopub_data_rate_limit=5000000" >> $JPY_CONFIG
```
Comment (etejedor) [inline systemuser.sh:51]: What is this for?

File: systemuser.sh [hunk 2]
```diff
@@ -84,6 +85,14 @@ sed -i "s/python/python$PYVERSION/g" $KERNEL_DIR/root/kernel.json # Set Python v
 # R
 cp -rL $LCG_VIEW/share/jupyter/kernels/ir $KERNEL_DIR
 sed -i "s/IRkernel::main()/options(bitmapType='cairo');IRkernel::main()/g" $KERNEL_DIR/ir/kernel.json # Force cairo for graphics
+# Octave
+OCTAVE_KERNEL_PATH=$LCG_VIEW/share/jupyter/kernels/octave
+if [[ -d $OCTAVE_KERNEL_PATH ]];
+then
+   cp -rL $OCTAVE_KERNEL_PATH $KERNEL_DIR
+   export OCTAVE_KERNEL_JSON=$KERNEL_DIR/octave/kernel.json
```
Comment (etejedor) [inline systemuser.sh:93]: Do we need this variable to be exported?

**Review by etejedor — CHANGES_REQUESTED** [review]

File: systemuser.sh (hunk 2 above)
Comment (diocas) [reply systemuser.sh:93]: Yes. This is the variable that the kernel picks to find the json file. (this was the PR I did to them)

File: systemuser.sh (hunk 1 above)
Comment (diocas) [reply systemuser.sh:51]: So that the kernel can send bigger images to the interface.

**Pull request merged**